src/CBT.Core/Tasks/WriteItemToJsonFile.cs

```diff
@@ -38,6 +38,12 @@ public bool Execute()
             }
             PackageRestoreData assetsFile = new PackageRestoreData
             {
+                ProjectJsonPath = Input
```

Comment:
unless the msbuild target that writes this file is overridden by someone or fails we should.  But I have gone ahead and handled the null just in case.